WC_Layers/config.ini

```diff
@@ -0,0 +1,43 @@
+[general]
+# Specify whether this problem uses structured or unstructured mesh
+structured: False
+# Specify whether this problem uses sub-voxel r2s
+sub_voxel: False
+
+[step1]
+# Path to MCNP MESHTAL file containing neutron fluxes or a DAG-MCNP5
+# unstructured mesh tally .h5m file.
+meshtal: Mesh_with_Tally.h5m
+# Tally number within the meshtal file containing the fluxes for activation.
+tally_num: 2
+# The name of the tag used to store flux data on the mesh. For unstructured
+# mesh this tag must already exist within the file specified in <meshtal>.
+flux_tag: FLUX_MESH
+# Path to the DAGMC material-laden geometry file (.h5m).
+geom: DAGMC_Geometry.h5m
```

Comment:
Does this geometry have materials in it?  Where is the script that does that?

Reply:
Yes, this is material laden. PyNE_Lib.py creates a PyNE material library which is then assigned to the DAG-MC geometry using the uwuw_preproc tool (done inside a terminal).